Sat Jan 22 14:21:26 EST 2022
Slurm nodes: evc6
You were assigned 1 gpu(s)
Please run `conda env list` to see a list of all available environments. Use
`source activate <env>` to activate the environment '<env>'. 

Currently Loaded Modules:
  1) anaconda/anaconda3

 

	[4mGPU0	mlx5_0	CPU Affinity	NUMA Affinity[0m
GPU0	 X 	SYS	8,10,12,14	0-1
mlx5_0	SYS	 X 		

Legend:

  X    = Self
  SYS  = Connection traversing PCIe as well as the SMP interconnect between NUMA nodes (e.g., QPI/UPI)
  NODE = Connection traversing PCIe as well as the interconnect between PCIe Host Bridges within a NUMA node
  PHB  = Connection traversing PCIe as well as a PCIe Host Bridge (typically the CPU)
  PXB  = Connection traversing multiple PCIe bridges (without traversing the PCIe Host Bridge)
  PIX  = Connection traversing at most a single PCIe bridge
  NV#  = Connection traversing a bonded set of # NVLinks


compress_rate: 0.5
Namespace(add_bn=True, arch='wideresnet28x10', baseline=True, checkpoint='cifar/checkpoint', compress_rate=0.5, dataset='cifar10', epochs=120, gamma=0.1, jobid='123379', l1_weight=0, l2_weight=0.001, logs='cifar/logs', lr=0.01, manualSeed=2261, momentum=0.9, schedule=[30, 60, 90], test_batch=100, train_batch=128, weight_decay=0.0005, workers=4)
Traceback (most recent call last):
  File "run_cifar.py", line 130, in <module>
    main()
  File "run_cifar.py", line 104, in main
    model = get_cifar_models(args.arch, args.dataset, pretrained=True)
  File "/lustre/fs0/home/mtayyab/codes/CNN-compression-ICIP2021/cifar/models/models.py", line 14, in get_cifar_models
    model = get_cifar10_models(model_name, pretrained)
  File "/lustre/fs0/home/mtayyab/codes/CNN-compression-ICIP2021/cifar/models/models.py", line 48, in get_cifar10_models
    model = wideresnet28x10(pretrained)
  File "/lustre/fs0/home/mtayyab/codes/CNN-compression-ICIP2021/cifar/models/wideresnet.py", line 92, in wideresnet28x10
    checkpoint = torch.load('wideresnet28x10_cifar10.pth')
  File "/apps/anaconda/anaconda3/envs/pytorch-gpu/lib/python3.6/site-packages/torch/serialization.py", line 382, in load
    f = open(f, 'rb')
FileNotFoundError: [Errno 2] No such file or directory: 'wideresnet28x10_cifar10.pth'

real	0m5.217s
user	0m2.068s
sys	0m0.495s

Ending script...
Sat Jan 22 14:21:33 EST 2022
